### PR TITLE
TC_INITSCRIPT support absolute/relative path

### PR DESCRIPTION
**Pull request by basicfu**
see #850 
TC_INITSCRIPT Add two path way
- begin with `root:` prefix,relative root path
- use absolute path

### CODE REVIEW

**Comment by basicfu** [comment]
Hi @kiview, I have already modified

File: modules/jdbc/src/main/java/org/testcontainers/jdbc/ContainerDatabaseDriver.java [hunk 1]
```diff
@@ -177,8 +179,15 @@ private void runInitScriptIfRequired(final ConnectionUrl connectionUrl, Database
         if (connectionUrl.getInitScriptPath().isPresent()) {
             String initScriptPath = connectionUrl.getInitScriptPath().get();
             try {
-                URL resource = Thread.currentThread().getContextClassLoader().getResource(initScriptPath);
-
+                URL resource;
+                if (initScriptPath.startsWith(FILE_PATH_PREFIX)) {
+                    //relative project path
+                    String relativePath = System.getProperty("user.dir") + "/" + initScriptPath.substring(FILE_PATH_PREFIX.length());
```
Comment (bsideup) [inline modules/jdbc/src/main/java/org/testcontainers/jdbc/ContainerDatabaseDriver.java]: why are you using `user.dir` here?
Also, why not just `return new URL(initScriptPath)`? :)

Comment (basicfu) [reply modules/jdbc/src/main/java/org/testcontainers/jdbc/ContainerDatabaseDriver.java]: use `user.dir` Use the project path

**Comment by rnorth** [comment]
I'll pick this up - will remove `user.dir` and add docs.

**Comment by rnorth** [comment]
Sorry, I didn't mean to close this PR. I've reopened as #1329 with a couple of tweaks.

**Pull request closed without merging**